### PR TITLE
Fix spellings in MergeTree and Sequence

### PR DESCRIPTION
**Pull request by anthony-murphy**
There were a bunch of spelling errors in these packages, along with some unused exports. This PR is just cleaning that up. No functional changes.

### CODE REVIEW

**Review by heliocliu — APPROVED** [review]

File: packages/dds/merge-tree/src/client.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ export class Client {
     }
 
     /**
-     * Annotate a maker and call the callback on concensus.
+     * Annotate a maker and call the callback on consensus.
```
Comment (scarlettjlee) [inline packages/dds/merge-tree/src/client.ts:129]: "marker" instead of "maker"

File: packages/dds/sequence/src/sharedString.ts [hunk 2]
```diff
@@ -30,7 +28,7 @@ export interface ISharedString extends SharedSegmentSequence<SharedStringSegment
      * Inserts the text at the position.
      * @param pos - The position to insert the text at
      * @param text - The text to insert
-     * @param props - The properties of text
```
Comment (pleath) [inline packages/dds/sequence/src/sharedString.ts:33]: Should this really be "text"?

Comment (anthony-murphy) [reply packages/dds/sequence/src/sharedString.ts:33]: yeah. not sure how that changed

File: packages/dds/sequence/src/sharedString.ts [hunk 3]
```diff
@@ -30,7 +28,7 @@ export interface ISharedString extends SharedSegmentSequence<SharedStringSegment
...
+     * @param props - The properties of tex
```
Comment (anthony-murphy) [inline packages/dds/sequence/src/sharedString.ts:31]:
```suggestion
     * @param props - The properties of text
```

File: packages/dds/sequence/src/intervalCollection.ts [hunk 4]
```diff
@@ -300,24 +294,6 @@ export class SequenceInterval implements ISerializableInterval {
         return (endPos > bstart) && (startPos < bend);
```
Comment (scarlettjlee) [inline packages/dds/sequence/src/intervalCollection.ts:294]: I'm surprised you didn't fix this too. bStart and bEnd.

**Review by scarlettjlee — APPROVED** [review]

**Review by pleath — APPROVED** [review]

File: packages/dds/merge-tree/src/client.ts (hunk 1 above)
Comment (anthony-murphy) [reply packages/dds/merge-tree/src/client.ts:129]:
```suggestion
     * Annotate a marker and call the callback on consensus.
```

File: packages/dds/sequence/src/intervalCollection.ts (hunk 4 above)
Comment (anthony-murphy) [reply packages/dds/sequence/src/intervalCollection.ts:294]: i have a bunch of things added to my user dictionary. i try not to be too strict about var names

File: packages/dds/matrix/src/permutationvector.ts [hunk 5]
```diff
@@ -282,9 +282,9 @@ export class PermutationVector extends Client {
         // containing this position and use 'findReconnectionPosition' to adjust for the local ops that
         // have not yet been submitted.
 
-        // eslint-disable-next-line max-len
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion, @typescript-eslint/no-unnecessary-type-assertion
-        return this.findReconnectionPostition(containingSegment, localSeq) + containingOffset!;
+
```
Comment (tylerbutler) [inline packages/dds/matrix/src/permutationvector.ts:285]: I hate this eslint bug... it leaves empty lines everywhere...

Comment (ChumpChief) [reply packages/dds/matrix/src/permutationvector.ts:285]: Once I get the definitions and server version bumps done i'll get the eslint fix in, which will add back the rule to catch double-blank-lines

**Pull request merged**